src/reducers/palettesReducer/palettesReducer.test.js

```diff
@@ -58,7 +58,7 @@ describe('palettesReducer', () => {
       }
     ];
     const result = palettesReducer([expected[0]], actions.addPalette(expected[1]));
-    expect(result).toEqual(expected);
+    expect(result.length).toEqual(2);
   });
```

Comment:
I think we should change this toEqual(expected.length) so it's move obvious about what we're checking.